### ISSUE TITLE
Sending event without ttl shows as 0 in stdout

### ISSUE THREAD

**Issue by alexforrow** [borntyping/python-riemann-client#19]
If you you don't include the --ttl option when sending an event using the cli, it correctly sends a nil value to Riemann, however it's shown as 0 on stdout.

The following give identical output on stdout but have different meanings in Riemann:
```
[alex@seventy riemann]$ riemann-client send --service test --ttl 0
{
  "attributes": {},
  "description": "",
  "host": "seventy",
  "metric_d": 0,
  "metric_f": 0,
  "metric_sint64": 0,
  "service": "test",
  "state": "",
  "tags": [],
  "time": 0,
  "ttl": 0
}
[alex@seventy riemann]$ riemann-client send --service test
{
  "attributes": {},
  "description": "",
  "host": "seventy",
  "metric_d": 0,
  "metric_f": 0,
  "metric_sint64": 0,
  "service": "test",
  "state": "",
  "tags": [],
  "time": 0,
  "ttl": 0
}
```

**Comment by borntyping** [comment]
This happens because Protocol Buffer messages return default values on access if a field is not set (i.e. 0 for numeric fields). I've modified the client to ignore unset fields when creating a dictionary, so output now only includes fields that were actually sent to Riemann: 

```
$ riemann-client send         
{
  "host": "centurion"
}
```
```
$ riemann-client send --ttl 10
{
  "host": "centurion", 
  "ttl": 10
}
```

**Comment by borntyping** [comment]
This is fixed in https://github.com/borntyping/python-riemann-client/commit/cf301356b71f33359ec90fce80df8a18dfab8d3b, and released by [version 6.0.0](https://warehouse.python.org/project/riemann-client/6.0.0/).

**Comment by alexforrow** [comment]
Awesome, thanks!
